### ISSUE TITLE
Feature Request: Ability to "lock" upterm session via GitHub secret

### ISSUE THREAD

**Issue by patcon** [lhotari/action-upterm#8]
This is a really next github action! Thanks! It def saved my butt :)

Anyhow, wondering if there's a way to make the session password-protected in some way, so that it's not accessible (complete with any secrets present in session) from anyone following the CI run logs.

Thanks for considering!

**Comment by patcon** [comment]
A-ha, found my answer! https://github.com/owenthereal/upterm/issues/72

Seems that I could use `--authorized-key=path/to/authorized_key` file. And as part of this action, it could generate at the very minimal the key of the person who committed, and maybe even the keys of reviewers and assignees :)

**Comment by lhotari** [comment]
@patcon This feature is now implemented. Please check the README

**Comment by patcon** [comment]
Thanks @lhotari! 🎉

**Comment by Folcon** [comment]
Hey,

I'm not sure if the docs are unclear or something is wrong, but I couldn't use either `limit-access-to-actor` together with `limit-access-to-users` or `limit-access-to-actor` by itself.

The ssh command I used was:
```shell
ssh -i ~/.ssh/gh_current_key b9mSRLZzh0Br3aVNmSNE:MTAuMjQ0LjAuNzY6MjI=@uptermd.upterm.dev
b9mSRLZzh0Br3aVNmSNE:MTAuMjQ0LjAuNzY6MjI=@uptermd.upterm.dev: Permission denied (publickey).
```

I also tried stripping the password string, as the `connection string` instruction in the docs is a little ambiguous.
```shell
ssh -i ~/.ssh/gh_current_key b9mSRLZzh0Br3aVNmSNE@uptermd.upterm.dev 
Connection closed by 157.230.199.75 port 22
```

To sanity check I tried doing an ssh connection test to github, which worked fine:
```
ssh -T -i ~/.ssh/gh_current_key git@github.com 
Warning: remote port forwarding failed for listen port 2222
Hi Folcon! You've successfully authenticated, but GitHub does not provide shell access.
```

Not sure what the problem is or if this is an upstream issue...
